Library/Homebrew/brew.sh

```diff
@@ -179,6 +179,15 @@ then
   set -- "$@" -v
 fi
 
+# Check if user passes --help command which should prevent auto-update.
+for arg in "$@"
+do
+  if [[ $arg = "--help" ]]
+  then
+    export HOMEBREW_COMMAND_HELP="1"
```

Comment:
Since it only persists for the duration of the subshell, should we just set `HOMEBREW_NO_AUTO_UPDATE` instead of adding another variable?